application/modules/admin/views/scripts/shipment/index.phtml

```diff
@@ -153,23 +143,7 @@
 <script type="text/javascript" charset="utf-8">
     /* Table initialisation */
     var oTable = null;
-    var duplicated = false;
 
-    function checkDuplicate(tableName, fieldName, obj, fnct, msg) {
-        if (obj.val() == "" || obj.val() == null) {
-            return;
-        }
-        $.post("<?php echo $this->url(array('module' => 'default', 'controller' => 'common', 'action' => 'check-duplicate')); ?>", {tableName: tableName, fieldName: fieldName, value: obj.val(), fnct: fnct, format: "html"},
-        function (data) {
-            if (data > 0) {
-                alert(msg, "err");
-                duplicated = true;
-                obj.focus();
-            } else {
-                duplicated = false;
-            }
-        });
-    }
 
```

Comment:
@brichardssa The duplication of the scheme code is added for the S1EPT-22 where we were checking the duplication of the scheme code. which was accidently left in this.

Reply:
Ok thanks. Please revert accidental changes so that we can try and review each ticket in isolation?

Reply:
@brichardssa .k we will try our best.

Reply:
@brichardssa The code is fine and ready for review in commit 3263562. The code was removed for keeping the relevant changes thanks. You can proceed further for review.

Reply:
@khadimfirdosi am I missing something? The check duplicate logic is still missing. I asked "Why did you remove this function?" and you replied that this was a leaking change from another ticket. I would like this part of the change to be reverted so that we don;t end up with an regression in the system that allows duplicated shipments from being created.

Reply:
The missing function is an unnecessary part for S1EPT-21 as that was written for S1EPT-22.

Reply:
@khadimfirdosi if we merge this code, we will be removing the check duplicate functionality that was added to the code in 2013 not removing a feature from S1EPT-22